ios/viewmodels/view-model-example.md

```diff
@@ -0,0 +1,280 @@
+# View Model Example
+
+This file lays out an example of view model use from the Prezto project.
+
+Prezto is an app used to create and share slideshows (preztos) directly on your mobile device.
+
+For this example, we show the implemenation of the "saved preztos" screen, which allows a user to:
+
+- Create a new prezto
+- View a list of existing prezto's
+- Delete a prezto
+
+![SavedPreztosViewController](prezto.png)
+
+NOTE: While this page does show code used in an actual project, the snippets here show only the relevant parts to illustrate view model use. The full versions of these files can be found here:
+
+- [`SavedPreztosViewModel.swift`](https://github.com/IntrepidPursuits/prezto-ios/blob/kevin/saved-preztos/prezto-ios/Features/SavedPreztos/SavedPreztosViewModel.swift)
+- [`SavedPreztoViewModel.swift`](https://github.com/IntrepidPursuits/prezto-ios/blob/kevin/saved-preztos/prezto-ios/Features/SavedPreztos/SavedPreztoViewModel.swift)
+- [`SavedPreztosViewController.swift`](https://github.com/IntrepidPursuits/prezto-ios/blob/kevin/saved-preztos/prezto-ios/Features/SavedPreztos/SavedPreztosViewController.swift)
+- [`SavedPreztoTableViewCell.swift`](https://github.com/IntrepidPursuits/prezto-ios/blob/kevin/saved-preztos/prezto-ios/Features/SavedPreztos/SavedPreztoTableViewCell.swift)
+
+## Parent View Model
+
+The `SavedPreztosViewController` contains a `SavedPreztosViewModel` that provides the backing logic for fetching and deleting prezto's. 
+
+The relevant portion is shown below:
+
+### SavedPreztosViewController.swift
+```swift
+class SavedPreztosViewModel {
+
+    var savedPreztos = [Prezto]() 
```

Comment:
Shouldn't this be `private`?  I can't see a great reason for a ViewModel to expose a list of Models publicly.